web/impact/impact/tests/test_graphql.py

```diff
@@ -4,7 +4,7 @@
 from django.urls import reverse
 
 from accelerator.models import (
-    UserRole,
+    UserRole, StartupRole
```

Comment:
style note: imports should be on separate lines

Reply:
Taken care of